sources/packages/api/src/services/application/application.service.ts

```diff
@@ -53,4 +56,29 @@ export class ApplicationService extends RecordDataModelService<Application> {
       .getOne();
     return application;
   }
+
+  async updateAssessmentInApplication(
+    applicationId: number,
+    assessment: any,
+  ): Promise<UpdateResult> {
+    return this.repo.update(applicationId, { assessment });
+  }
+
+  async getAssessmentByApplicationId(
+    applicationId: number,
+  ): Promise<ApplicationAssessmentDTO> {
+    const application = await this.repo
+      .createQueryBuilder("application")
+      .select("application.assessment")
+      .where("application.id = :applicationId", { applicationId })
+      .getOne();
+    return {
+      weeks: application.assessment.weeks,
```

Comment:
We should not be expressing these as they are supposed to be dynamic and flexible.

Reply:
Do you want to return any? i.e. the assessment object itself?